cloudbuild.yaml

```diff
@@ -1,21 +1,21 @@
 steps:
-  - name: 'gcr.io/cloud-builders/docker'
-    args: ['build', '-t', 'gcr.io/$PROJECT_ID/$REPO_NAME:$COMMIT_SHA', '.']
-    timeout: '1500s'
-  - name: 'gcr.io/cloud-builders/docker'
-    args: ['push', 'gcr.io/$PROJECT_ID/$REPO_NAME:$COMMIT_SHA']
+  - name: 'gcr.io/kaniko-project/executor:latest'
+    args:
+      - --destination=gcr.io/$PROJECT_ID/$REPO_NAME:latest
+      - --cache=true
+      - --dockerfile=Dockerfile
+    id: build
```

Comment:
No need for timeout anymore?

Reply:
I'm not sure. One would think that a global timeout would let each step run until the global deadline is met. I'll go read the docs to make sure!

Reply:
Looks like it does:

> Use the timeout field in a build step to set a time limit for executing the step. If you don't set this field, the step has no time limit and will be allowed to run until either it completes or the build itself times out. The timeout field in a build step must not exceed the timeout value specified for a build. timeout must be specified in seconds with up to nine fractional digits, terminated by 's'. Example: "3.5s"